opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/entitySetting-test.js

```diff
@@ -60,7 +60,8 @@ describe('EntitySetting resolver standard behavior', () => {
     const context = executionContext('test');
     await initCreateEntitySettings(context);
     const queryResult = await queryAsAdmin({ query: LIST_QUERY });
-    expect(queryResult.data.entitySettings.edges.length).toEqual(34);
+    // Updated because we have add two entitySettings
+    expect(queryResult.data.entitySettings.edges.length).toEqual(36);
```

Comment:
2 more types, but Case souldn't be returned
It's like an interface, no ?

![image](https://user-images.githubusercontent.com/53513584/229510269-c1204b0f-cdad-4d13-86a5-d030c774e0e6.png)